GoogleForCreators/web-stories-wp
# Canvas: Video poster not rendering in Safari

## Context



## Summary
While working on #11849, I found a quick fix for this issue. Setting background image for poster seems to work well here.  

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

This issues an issue in safari. Test in all browser but with a special focus safari. 
1. Create story. 
2. Insert video onto canvas. 
3. Add a custom video poster ( both upload file and link to file ). 
4. See poster render. 
5. Create another page, change to new page. 
6. Go back to original page. See poster. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10167
